starter-code/src/App.js

```diff
@@ -1,18 +1,113 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import logo, { ReactComponent } from "./logo.svg";
+import "./App.css";
+import contacts from "./contacts.json";
+const firstFive = [];
+for (let i = 0; i < 5; i++) {
+  firstFive.push(contacts[i]);
+}
+//console.log(contacts.length);
+
+//console.log(randomNumber);
+class Table extends Component {
+  state = {
+    firstContacts: firstFive
+  };
+  add = () => {
+    let randomNumber = Math.floor(Math.random() * 199 + 5);
+    this.state.firstContacts.push(contacts[randomNumber]);
+    this.setState({
+      firstContacts: this.state.firstContacts
+    });
+  };
+
+  sortByName = () => {
+    this.setState({
+      firstContacts: this.state.firstContacts.sort((a, b) => {
+        if (a.name.toLowerCase() < b.name.toLowerCase()) {
```

Comment:
If they are equally bit it is supposed to return 0. localeCompare is the best practice for comparing strings though